test/expected/normalized.out

```diff
@@ -62,28 +59,28 @@ Referenced by:
     TABLE "input_values" CONSTRAINT "input_values_labels_id_fkey" FOREIGN KEY (labels_id) REFERENCES input_labels(id)
 
 \d+ input_copy
-                        Table "public.input_copy"
- Column |    Type     | Modifiers | Storage | Stats target | Description 
---------+-------------+-----------+---------+--------------+-------------
- sample | prom_sample | not null  | plain   |              | 
+                                  Table "public.input_copy"
+ Column |    Type     | Collation | Nullable | Default | Storage | Stats target | Description 
```

Comment:
What is the point of all of these tests that do \d? Are we really concerned about that remaining the same? This seems brittle on output of \d and also like the tests are there to make us feel like we're testing things, but not actually adding much value, can we test what we actually want to test here?

Reply:
This might not be needed for this commit, but is a comment more generally I think we should think about here.

Reply:
Well this was not really part of my change(I only adjusted formatting) but I can explain it as making sure that DB schema is correct, so I do believe that there is some value here.
I agree that this output seems brittle and as we see it might change between PostgreSQL versions. I guess alternative is to have our own table describe function and we can maybe tackle that in another PR.